downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -95,3 +95,48 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 ### Bug fixes
 
 ## Backward-incompatible changes
+
+### Ballerina library changes
+
+#### `jwt` package
+
+- Add support to directly provide `crypto:PrivateKey` and `crypto:PublicKey` in JWT signature configurations. With this update, the `config` field in `jwt:IssuerSignatureConfig` now supports `crypto:PrivateKey`, and the `certFile` field in `jwt:ValidatorSignatureConfig` now supports `crypto:PublicKey`. These additions will breaks the previous union-type support.
+    ```ballerina
+    // previous `jwt:IssuerSignatureConfig` record
```

Comment:
Capitalize the first letter and add a period at the end.

Reply:
Fixed with [89db88c9172a7a8cfc4671b2ef083645bbb74c7b](https://github.com/ballerina-platform/ballerina-dev-website/pull/9221/commits/89db88c9172a7a8cfc4671b2ef083645bbb74c7b)